src/Outsystems.SetupTools/Lib/Constants.ps1

```diff
@@ -194,6 +194,8 @@ $OSIISConfig = @(
         'Match'            = @('/LT*', '/lifet*', '/LifeT*', 'PerformanceMonitor')
     }
 )
+[System.Diagnostics.CodeAnalysis.SuppressMessage('PSUseDeclaredVarsMoreThanAssigments', '')]
```

Comment:
you need to rebase your branch, these changes are already merged and published in the latest version of SetupTools